README.md

```diff
@@ -92,12 +92,11 @@ types, groups and group content given here are suggestions and are given to
 provide a quick way to get started with Organic groups.
 
 1. Enable the Group and the Group UI modules.
-2. Create a new content type via admin/structure/types/add. Call it "Group", and
-   define it to be of Group type.
-3. Create a second content type. Call it "Group content", and set it to be of
-   Group content type.
-4. Add a Group by going to node/add/group. Call it First group, and enable the
-   Group through the "Group type" field.
+2. Create a new content type via admin/structure/types/add. Call it "Group",
+   and click on tab "Organic group" then define it to be of Group type.
+3. Create a second content type. Call it "Group content", and click on tab 
+   "Organic group" then set it to be of Group content type.
```

Comment:
This tab is called "Organic groups" with an "s".

For the rest this looks good, thanks!